tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -275,6 +279,25 @@ 1. IWA
                             }
 
                             break;
+
+                        case 12: // acquire token interactive with WamBroker
```

Comment:
As a team we should clarify which broker we're referring to. I propose to explicitly state 'legacy' for anything related to the old broker.

Reply:
Hmm, there really should not be a "legacy" broker. We only use the new broker. Only on UWP we use the "legacy" broker because the new broker does not support UWP.

Reply:
Case 12 is for usign the old/legacy WamBroker